test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -339,11 +376,8 @@ namespace iroha {
                        .getAccount("some@domain")
                        .build();
       auto result = executeQuery(query);
-      ASSERT_TRUE(boost::apply_visitor(
-          shared_model::interface::QueryErrorResponseChecker<
-              shared_model::interface::NoAccountErrorResponse>(),
-          result->get()))
-          << result->toString();
+      checkStatefulError<shared_model::interface::NoAccountErrorResponse>(
+          std::move(result), 0);
```

Comment:
Can we at add constants for the test? At least local variables with good naming?

Reply:
OK, constants for test only can be added, but they are to be substituted by the solution for named constants of both Command and Query Executors